ROMFS/px4fmu_common/init.d/2200_standard_vtol_hitl

```diff
@@ -0,0 +1,80 @@
+#!nsh
+#
+# @name Standard vtol HITL
+#
+# @type Standard VTOL
+# @class VTOL
+#
+# @maintainer Roman Bapst <roman@auterion.com>
+#
+
+sh /etc/init.d/rc.vtol_defaults
+
+if [ $AUTOCNF == yes ]
+then
+	param set BAT_N_CELLS 3
+	param set CAL_ACC0_ID 1376264
+	param set CAL_ACC0_XOFF 0.01
+	param set CAL_ACC0_XSCALE 1.01
```

Comment:
Why are you setting the calibration parameters?

Reply:
@dagar This is probably coming from the SITL startup file. I'll go over the params and kick out the unnecessary ones.